common.go

```diff
@@ -8,11 +8,12 @@ import (
 	"gopkg.in/src-d/go-git.v4/core"
 )
 
-// Storage storage of objects and references
-type Storage interface {
-	ConfigStorage() config.ConfigStorage
-	ObjectStorage() core.ObjectStorage
-	ReferenceStorage() core.ReferenceStorage
+// Storer storage of objects, references and all the information that require
```

Comment:
This sentence makes no sense, it has no verb.

Also, is this comment accurate?, does the memory implementation persist data?